src/routes/github/subscription/github-subscription-delete.ts

```diff
@@ -1,82 +1,67 @@
-import { Subscription } from "../../../models";
 import { Request, Response } from "express";
+import { GitHubAPI } from "probot";
+import { Subscription } from "../../../models";
+import { getCloudInstallationId } from "../../../github/client/installation-id";
+import { GitHubAppClient } from "../../../github/client/github-app-client";
+import { GitHubUserClient } from "../../../github/client/github-user-client";
+import { booleanFlag, BooleanFlags } from "../../../config/feature-flags";
```

Comment:
once you merge with main, you should change these paths to the new shortened ones :)